README.md

```diff
@@ -1,32 +1,202 @@
-# Accessor and Mutator Methods
+# Scope
```

Comment:
I think this is supposed to be about accessor and mutator methods - not scope?

Reply:
Yes, thanks!